src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public abstract class Figure {
+    private int area;
```

Comment:
we don't need this variable here, area would calculated by method getArea()

Reply:
done